app/controllers/api/v1/config_templates_controller.rb

```diff
@@ -78,9 +80,23 @@ def destroy
       api :GET, "/config_templates/build_pxe_default", "Change the default PXE menu on all configured TFTP servers"
 
       def build_pxe_default
-        status, msg = ConfigTemplate.authorized(:deploy_templates).build_pxe_default(self)
+        status, msg = ProvisioningTemplate.authorized(:deploy_provisioning_templates).build_pxe_default(self)
         render :json => msg, :status => status
       end
+
+      private
+
+      def type_name_singular
+        @type_name_singular ||= resource_name
+      end
+
+      def resource_class
+        ProvisioningTemplate
+      end
+
+      def deprecated
+        ::ActiveSupport::Deprecation.warn('Config templates were renamed to provisioning templates')
```

Comment:
right, that was also my conclusion but I'd like to keep it out of this PR, created an issue http://projects.theforeman.org/issues/10635